server/src/ResolveLocation.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * kalman.c
+ *
+ *  Created on: 23 nov 2017
+ *      Author: alexs
+ */
+
+
+#include <math.h>
+#include <stdint.h>
+#include "../include/ResolveLocation.h"
```

Comment:
Can we include this in a different way without specifying the relative path (i.e. include it via the CMake)?

Reply:
same goes for this.